pxr/base/tf/fileUtils.cpp

```diff
@@ -83,8 +83,7 @@ Tf_HasAttribute(
     if (path.back() == '/' || path.back() == '\\')
         resolveSymlinks = true;
 
-    const DWORD attribs =
-        GetFileAttributesW(ArchWindowsUtf8ToUtf16(path).c_str());
+    DWORD attribs = GetFileAttributesW(ArchWindowsUtf8ToUtf16(path).c_str());
```

Comment:
Minor nitpick, but since we re-use this, might be nice to save the Utf16 path...